### PR TITLE
WIP PLAT-139: appcontroller using insconfig

### PR DESCRIPTION
**Pull request by AndreyBronin**
**- What I did**

- Use [insconfig](https://github.com/insolar/insconfig) package for manage configuration
- AppController cobra commands for all described modes
- Fix launchnet.sh script 

**- How to verify it**
launchnet.sh -g

### CODE REVIEW

File: ledger-core/v2/cmd/insolard/commands.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package main
+
+import (
+	"github.com/insolar/assured-ledger/ledger-core/v2/log/global"
```
Comment (sexyhamster) [inline ledger-core/v2/cmd/insolard/commands.go]: wrong import order, you need to copy IDE config

File: ledger-core/v2/configuration/logicrunner.go [hunk 2]
```diff
@@ -13,16 +13,13 @@ type LogicRunner struct {
 	// e.g. "tcp", "unix"... see `net.Dial`
 	RPCProtocol string
 	// BuiltIn - configuration of builtin executor
-	BuiltIn *BuiltIn
+	BuiltIn bool
```
Comment (ruz) [inline ledger-core/v2/configuration/logicrunner.go]: This shouldn't be converted to bool. Delete this option.

File: ledger-core/v2/scripts/generate_insolar_configs.go [hunk 3]
```diff
@@ -247,8 +247,11 @@ var defaultInsloardConf *configuration.Configuration
 
 func newDefaultInsolardConfig() configuration.Configuration {
 	if defaultInsloardConf == nil {
-		holder := configuration.NewHolderWithFilePaths(insolardDefaultsConfig).MustInit(true)
-		defaultInsloardConf = &holder.Configuration
+		//holder := configuration.NewHolder(insolardDefaultsConfig)
+		//holder.MustLoad()
+		//defaultInsloardConf = holder.Configuration
```
Comment (ruz) [inline ledger-core/v2/scripts/generate_insolar_configs.go]: commented out code shouldn't be in the repository

File: ledger-core/v2/logicrunner/logicrunner.go [hunk 4]
```diff
@@ -142,7 +142,7 @@ func (lr *LogicRunner) Start(ctx context.Context) error {
 		lr.rpc.Start(ctx)
 	}
 
-	if lr.Cfg.BuiltIn != nil {
+	if lr.Cfg.BuiltIn == true {
```
Comment (ruz) [inline ledger-core/v2/logicrunner/logicrunner.go]: I'm ok with this condition deleted.

File: ledger-core/v2/configuration/configuration.go [hunk 5]
```diff
@@ -7,14 +7,13 @@ package configuration
 
 import (
 	"fmt"
-	"path/filepath"
-	"reflect"
-	"strings"
+	"github.com/insolar/insconfig"
```
Comment (sexyhamster) [inline ledger-core/v2/configuration/configuration.go]: wrond imports order

File: ledger-core/v2/cmd/insolard/commands.go (hunk 1 above)
Comment (AndreyBronin) [reply ledger-core/v2/cmd/insolard/commands.go]: order correct in this case

Comment (sexyhamster) [reply ledger-core/v2/cmd/insolard/commands.go]: github.com/insolar/assured-ledger is local, so it should be after external dependency

see example in 
ledger-core/v2/cmd/insolard/main.go

File: ledger-core/v2/configuration/configuration.go (hunk 5 above)
Comment (AndreyBronin) [reply ledger-core/v2/configuration/configuration.go]: goimport file watcher said it's correct.
Don't argue with robot

File: ledger-core/v2/cmd/insolard/commands.go (hunk 1 above)
Comment (AndreyBronin) [reply ledger-core/v2/cmd/insolard/commands.go]: goimport said - Nope

File: ledger-core/v2/configuration/configuration.go [hunk 6]
```diff
@@ -7,14 +7,52 @@ package configuration
 
 import (
 	"fmt"
-	"path/filepath"
-	"reflect"
-	"strings"
 
-	"github.com/spf13/viper"
+	"github.com/insolar/insconfig"
```
Comment (aalekseev-ru) [inline ledger-core/v2/configuration/configuration.go]: this import order seems wrong too

Comment (aalekseev-ru) [inline ledger-core/v2/configuration/configuration.go]: this import order seems wrong too

Comment (AndreyBronin) [reply ledger-core/v2/configuration/configuration.go]: nope, its correct

**Pull request merged**